app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -40,7 +40,8 @@ def destroy
 
     def apply_secure_headers_override
       override_content_security_policy_directives(
-        form_action: ["'self'", @authorize_form.allowed_form_action].compact
+        form_action: ["'self'", @authorize_form.allowed_form_action].compact,
+        preserve_schemes: true
```

Comment:
I think @zachmargolis is saying, we delegate the authority to claim a `http` scheme is legitimate to either (a) the git repo via `service_providers.yml` or to the identity dashboard, which has its own authentication protocol. So it's legit for `http` to exist in production because we have chosen to trust the dashboard.